modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,36 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$disabled_functions = explode( ',', ini_get( 'disable_functions' ) );
```

Comment:
Hey @georgestephanis. Thanks for all the feedback.

Firstly, about "rare server configuration". As I said before, Locaweb is one of the most common hosting providers in Brazil, and some of accounts are configured with ```print_r``` disabled. It is possible that other people will experience this same problem.

Regarding the use of the filter, it is really a better strategy...
```php
$all_values_string = apply_filters( 'grunion_print_all_values', @print_r( $all_values, true ), $all_values );
```

I did not really know that ```function_exists``` detected disabled functions, so because of your hint I've switched to:

```php
if ( function_exists( 'print_r' ) ) {
   // ...
}
```